Add sections about customizing user settings to user manual (#1495)
Uninett/Argus: docs/user-manual.rst

3. Select the preferred theme from the drop-down menu.

Add your own theme
~~~~~~~~~~~~~~~~~~

> There are 3 themes available out of the box in Argus: *Argus*, *Dark* and *Light*. You can
> add your own theme by `overriding the default DAISYUI_THEMES

Reviewer: ```suggestion
There are three themes available out of the box in Argus: *Argus*, *Dark* and
*Light*. You can add your own theme by `overriding the default DAISYUI_THEMES
```

or create your own. For creating your own theme, we recommend using the
`daisyUI theme generator <https://v4.daisyui.com/theme-generator/>`_.

Change date and time format
~~~~~~~~~~~~~~~~~~~~~~~~~~~

1. `Open user menu`_.

> 2. Click on the *Date format* drop-down under the *User preferences*.

Reviewer: ```suggestion
2. Click on the *Date format* drop-down in the *User preferences* section.
```

3. Select the preferred date format from the drop-down menu.
   The date format will be applied to all dates in Argus.
   Available options are:

    -  ``LOCALE`` - uses the date format of your browser locale.
    -  ``ISO`` - uses the ISO date format (YYYY-MM-DD HH:MM:SS).
    -  ``RFC`` - uses the RFC date format (Day, DD Mon YYYY HH:MM:SS GMT).

>     -  ``EPOCH`` - uses the Unix epoch date format (seconds since 1970-01-01 00:00:00 UTC).

Reviewer: ```suggestion
    -  ``EPOCH`` - uses the Unix epoch date format (seconds since
       1970-01-01 00:00:00 UTC).
```
